.pre-commit-config.yaml

```diff
@@ -25,3 +25,8 @@ repos:
     rev: 5.12.0
     hooks:
       - id: isort
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.292
+    hooks:
+      - id: ruff
```

Comment:
black and ruff both?

Reply:
ruff will lint, black will check formatting.

Reply:
Also, we ignore line length errors on ruff.